Gaijinizer patch/Map003.txt

```diff
@@ -1503,50 +1503,53 @@ the number of registrations.#MTLed
  本来なら、気力を最大まで強化しないと厳しいが
 \\C[2]今回は百鬼のアイテムのおかげで、初期気力でも使える。
 > CONTEXT: Map003/events/1/pages/2/389/Dialogue/クッション君
-And this one is stupid for consumption vitality.
-Originally, it is tough if you do not strengthen
-the vitality to the maximum. \\C[2]This time, thanks to
-Hyakki's items, you can use the initial vitality
-as well.#MTLed
+Also, the vitality consumption for this one is
+stupid high. Ordinarily, it would be difficult if
+you didn't upgrade vitality to the maximum, but...
+
```

Comment:
Tiny bit of cleanup:
>_**Also, the vitality consumption for this one is
stupid high. Normally it would be difficult to
use if you didn't upgrade vitality to the
maximum, but...
\\C[2]This time, thanks to Hyakki's items, you can
pull it off even with your starting vitality.**_